pkg/resource/apps-toggle/reconciler.go

```diff
@@ -40,31 +40,31 @@ func (r *Reconciler) GetObservabilityBundleVersion(ctx context.Context, lc logge
 	return semver.Parse(currentApp.Spec.Version)
 }
 
-// ReconcileCreate ensure Promtail is enabled in the given cluster.
+// ReconcileCreate ensure Promtail/GrafanaAgent is enabled in the given cluster.
 func (r *Reconciler) ReconcileCreate(ctx context.Context, lc loggedcluster.Interface) (ctrl.Result, error) {
 	logger := log.FromContext(ctx)
-	logger.Info("promtailtoggle create")
+	logger.Info("Promtail and GrafanaAgent toggle create")
 
 	observabilityBundleVersion, err := r.GetObservabilityBundleVersion(ctx, lc)
 	if err != nil {
 		return ctrl.Result{}, errors.WithStack(err)
 	}
 
-	// Get desired configmap to enable promtail.
+	// Get desired configmap to enable Promtail/GrafanaAgent.
 	desiredConfigMap, err := GenerateObservabilityBundleConfigMap(lc, observabilityBundleVersion)
```

Comment:
I would replace all Promtail/GrafanaAgent occurrences by logging agents tbh because if we add or remove an app it will bé a pain